### PR TITLE
Feature/lab3

### PR DESCRIPTION
**Pull request by ViktoriiaPartyka**
Lab3 added

### CODE REVIEW

**Comment by andrii-s-shpitser-lpnu-ua** [comment]
Where is tests?

**Comment by ViktoriiaPartyka** [comment]
I don’t  have tests as such. In a comment to 5-7 laboratory I explained the reason.

**Comment by andrii-s-shpitser-lpnu-ua** [comment]
It is not a reaon. You can add return status to function or check global data that were changed by your functions.

File: right values.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+// Unit test, right values
```
Comment (andrii-s-shpitser-lpnu-ua) [inline right values.cpp:2]: there is no test - just a sceletone.

Comment (ViktoriiaPartyka) [reply right values.cpp:2]: Sorry, I have exidently added the wrong file. 

I have fixed it now.

Comment (ViktoriiaPartyka) [reply right values.cpp:2]: Here is a commit for that fix: https://github.com/ViktoriiaPartyka/csdt-2122-ki48-ViktoriiaPartyka/pull/3/commits/53480dde4e53ba144f3c4bf59bba03870a221a43

Comment (andrii-s-shpitser-lpnu-ua) [reply right values.cpp:2]: I do not agree but I have no time to argue with you.

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Pull request merged**